app/uk/gov/hmrc/apiscope/services/ScopeJsonFileReader.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiscope.services
+
+import java.nio.file.{Files, Paths}
+
+import javax.inject.Singleton
+import play.api.Logger.logger
+
+@Singleton
+class ScopeJsonFileReader {
+
+  def readFile: Option[String] = {
```

Comment:
I had a go at inject Play's environment here but it seemed quite messy when I was left handling closing the reader in a finally block. What I've done here isn't very purely Scala or Play so I hope it's OK